proxywasm/v2/types.go

```diff
@@ -59,7 +59,7 @@ const (
 	MetricTypeHistogram MetricType = 3
 )
 
-type MapType int32
+type MapType = int32
```

Comment:
This allows wasmer to use the type alias as a parameter (wazero can use either way)